docs/sources/docker-hub/official_repos.md

```diff
@@ -1,189 +1,113 @@
-page_title: Guidelines for Official Repositories on Docker Hub
+page_title: Official Repositories on Docker Hub
 page_description: Guidelines for Official Repositories on Docker Hub
 page_keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, docs, official, image, documentation
 
-# Guidelines for Creating and Documenting Official Repositories
+# Official Repositories on Docker Hub
+
+You should know by now that a Docker repository (repo) is a collection of
```

Comment:
This "you should know now" wording assumes some linearity to the way that users are finding this document. I'd recommend more neutral language like "As discussed in X, a Docker repository (repo) is a collection of images."